Packs/PrismaAccess/Integrations/PrismaAccess/PrismaAccess.py

```diff
@@ -0,0 +1,470 @@
+
+    return json_result
+
+
+def prisma_access_test():
+    """
+    test module
+    """
+
+    # Test API connection only if user configured it
+    if apiConfigured:
+        try:
+            params = {
+                'type': 'op',
+                'cmd': '<show><system><info></info></system></show>',
+                'key': API_KEY
+            }
+
+            http_request(
+                URL,
+                'GET',
+                params=params
+            )
+
+            if DEVICE_GROUP and DEVICE_GROUP != 'shared':
+                device_group_test()
+        except Exception as ex:
+            raise type(ex)("PAN-OS XML API Test:\n" + str(ex))
+
+    # Test SSH connection only if user configured it
+    if sshConfigured:
+        try:
+            panos_connect()
+        except Exception as ex:
+            raise type(ex)("PAN-OS SSH CLI Test:\n" + str(ex))
+
+    # If reached this point, all ok
+    demisto.results('ok')
+
+
+def get_device_groups_names():
+    """
+    Get device group names in the Panorama
+    """
+    params = {
+        'action': 'get',
+        'type': 'config',
+        'xpath': "/config/devices/entry/device-group/entry",
+        'key': API_KEY
+    }
+
+    result = http_request(
+        URL,
+        'GET',
+        params=params
+    )
+
+    device_groups = result['response']['result']['entry']
+    device_group_names = []
+    if isinstance(device_groups, dict):
+        # only one device group in the panorama
+        device_group_names.append(device_groups.get('@name'))
+    else:
+        for device_group in device_groups:
+            device_group_names.append(device_group.get('@name'))
+
+    return device_group_names
+
+
+def device_group_test():
+    """
+    Test module for the Device group specified
+    """
+    device_group_names = get_device_groups_names()
+    if DEVICE_GROUP not in device_group_names:
+        raise Exception(f'Device Group: {DEVICE_GROUP} does not exist.'
+                        f' The available Device Groups for this instance: {", ".join(device_group_names)}.')
+
+
+@logger
+def prisma_access_logout_user(computer: str, domain: str, user: str) -> Dict[str, str]:
+    if apiConfigured:
+        xmlComputer = '<computer>%s</computer>' % b64encode(computer.encode('utf8')).decode('utf8') if computer else ''
+        b64User = (b64encode(user.encode('utf8'))).decode('utf8')
+        params = {
+            'type': 'op',
+            'key': API_KEY,
+            'cmd': '''<request><plugins><cloud_services><gpcs>
+                    <logout_mobile_user><gateway>%s<domain>%s</domain><user>%s</user></gateway></logout_mobile_user>
+                    </gpcs></cloud_services></plugins></request>''' % (xmlComputer, domain, b64User)
+        }
+        result = http_request(URL, 'GET', params=params)
+        return result
+    else:
+        return_error('You must configure the PAN-OS API Key and Port parameters to use this command.')
```

Comment:
instead of return_error here, `raise Exception('You must configure the PAN-OS API Key and Port parameters to use this command.')`, this way you will not get this lint error
![image](https://user-images.githubusercontent.com/7270217/80303648-2a34ed80-87ba-11ea-8199-be3a184966c2.png)